ionic-team/stencil
# chore(release): split prompts into two separate files

✔️  ~Depends On: https://github.com/ionic-team/stencil/pull/4235~ Landed
## Pull request checklist

Please check if your PR fulfills the following requirements:

## Pull request type

 

 

Please check the type of change your PR introduces:


## What is the current behavior?


GitHub Issue Number: N/A


## What is the new behavior?


Note: Each commit in this PR is a single step in refactoring this code out into two separate files. I'm avoiding a commit-by-commit description here, as I assume that this PR will be rebased at least once after its parent lands, and may have to take requested changes from the parent into account.

this commit splits the existing `prompts.ts` file into two separate files:
- `release-prepare-prompts` for release preparation prompts
- `release-prompts` for release prompts

this is being done to help with the navigation between the two, as suggested in https://github.com/ionic-team/stencil/pull/4173#discussion_r1142604610

## Does this introduce a breaking change?




## Testing


Check out this branch and perform the following:
 ⚠️ These steps must be followed closely, otherwise we risk a release getting out into the wild ⚠️ 
1. Run `npm run release.prepare -- --any-branch --dry-run` to evaluate that the release preparation scripts run as intended 
    - `--any-branch` is needed bc we're not on `main` 
    - `--dry-run` is needed to skip checks related to being on `main` and ensuring `main` is in sync with GitHub. It will still perform npm install, building, and validation of the build
2. select '3.2.1-0' when prompted for a version - this version should appear in `CHANGELOG.md`, `package.json`, and `package-lock.json` after the script runs.
these changes, as we'll use them in the next section.
3. Run `npm run release -- --any-branch --dry-run` to evaluate that the release preparation scripts run as intended 
    - `--any-branch` is needed bc we're not on `main` 
    - `--dry-run` is needed to skip the actual release
    - Select 'dev' as the tag (just in case)
    - Use 'hello!' as your OTP (just in case)
4. Your browser will open a GitHub Release form. The publish did _not_ occur, this step does not get skipped on `--dry-run`. Verify the version number in the form, and the tag used in STDOUT

## Other information
